packages/api/test/src/domain/bufficorn.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { Bufficorn } from '../../../src/domain/bufficorn'
+import { Trait, BufficornLeaderboardInfo } from '../../../src/types'
+
+describe('bufficorn.ts', () => {
+  it('getScore', async () => {
+    const bufficorn = new Bufficorn(undefined, 0)
+    expect(bufficorn.calculateScore()).toStrictEqual(0)
+    expect(bufficorn.calculateScore(Trait.Vigor)).toStrictEqual(0)
+
+    bufficorn.stats.vigor = 10
+    expect(bufficorn.calculateScore()).toStrictEqual(0)
+    expect(bufficorn.calculateScore(Trait.Vigor)).toStrictEqual(10)
+
+    bufficorn.stats.agility = 20
+    bufficorn.stats.coat = 30
+    bufficorn.stats.coolness = 40
+    bufficorn.stats.speed = 50
+    bufficorn.stats.stamina = 60
+    expect(bufficorn.calculateScore()).toStrictEqual(10)
+    expect(bufficorn.calculateScore(Trait.Vigor)).toStrictEqual(10)
```

Comment:
What do you think about splitting this test in two different to avoid test two different behaviours at one? I suggest test calculate score with trait and without trait.